main.cpp

```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Dog
+//  action 1: The dog barks.
+dog.bark();
+//  action 2: The dog sleeps.
+dog.sleep();
+//  action 3: The dog eats.
+dog.eat();
 
 //  2)
+//  Noun: Radio
+//  action 1: The radio receives signals.
+radio.receive();
+//  action 2: The radio transmits signals.
+radio.transmit();
+//  action 3: The radio changes frequencies.
+radio.setFrequency();
+
+//  3) musicPlayer
 //  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  action 1: The music player plays a song.
+musicPlayer.play();
+//  action 2: The music player stops playing a song.
+musicPlayer.stop();
+//  action 3: The music player opens a new file.
+musicPlayer.openFile();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Car 
+//  action 1: The car accelerates.
+car.accelerate();
+//  action 2: The car slows down.
+car.break();

```

Comment:
These function names should match the plain English nearly identically.
- [ ] revise